Content.Server/Shuttles/Components/ShuttleComponent.cs

```diff
@@ -26,6 +30,12 @@ public sealed partial class ShuttleComponent : Component
         [ViewVariables]
         public readonly float[] LinearThrust = new float[4];
 
+        /// <summary>
+        /// The cached thrust available for each cardinal direction, if all thrusters are T1
+        /// </summary>
+        [ViewVariables]
+        public readonly float[] BaseLinearThrust = new float[4];
```

Comment:
Is this supposed to be a datafield.

Reply:
I don't think so, like `LinearThrust` it's the cached running total of base thrust.

Reply:
Yep, just a cached total of base thrust, because the exact thrust factor can be recalculated with a lot less headache.